tutorials/script/core/9_pre_transitions_processing.py

```diff
@@ -35,14 +35,14 @@
 
 # %%
 def save_previous_node_response_to_ctx_processing(
-    ctx: Context, _: Pipeline, *args, **kwargs
+    ctx: Context, _: Pipeline
 ) -> Context:
```

Comment:
Collapse to one line?

Reply:
No, that would make lines longer than 80 characters, which would result in them not fitting into code blocks in the docs (#241).
An alternative is to rename these functions to have shorter names (e.g. `save_previous_node_response` and `prepend_previous_node_response`). This is done in d8075f0cab22bb43d5863dc17abd6519e3be97b9.